discord/enums.py

```diff
@@ -1153,6 +1152,16 @@ class SKUType(Enum):
     subscription        = 5
     subscription_group  = 6
 
+class EntitlementType(Enum):
```

Comment:
Please add this to the  `__all__` (public objects) tuple at the top of the file as well.

Reply:
oh, yeah, forgot it